web/src/components/CountryDropdown.tsx

```diff
@@ -59,14 +60,19 @@ export const CountryDropdown = (props: Props): ReactElement => {
       : countries;
 
   const getCountry = (value: string | undefined): CountriesObject | undefined => {
+    if (props.options) {
+      return props.options.find((country: CountriesObject) => {
+        return country.name === value || country.shortCode === value?.toUpperCase();
+      });
+    }
     return filteredCountries().find((country: CountriesObject) => {
       return country.name === value || country.shortCode === value?.toUpperCase();
     });
```

Comment:
Thanks!
Dust: could these two be merged? as in
```suggestion
      const options = props.options ?? filteredCountries()
      return options.find((country: CountriesObject) => {
        return country.name === value || country.shortCode === value?.toUpperCase();
      });
```